sources/packages/backend/libs/integrations/src/services/institution-location/institution-location.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from "@nestjs/common";
+import { InstitutionLocation, RecordDataModelService } from "@sims/sims-db";
+import { DataSource } from "typeorm";
+
+/**
+ * Institution location service.
+ */
+@Injectable()
+export class InstitutionLocationService extends RecordDataModelService<InstitutionLocation> {
+  constructor(dataSource: DataSource) {
```

Comment:
Can we use the @InjectRepository for the new classes and stop using `extends RecordDataModelService`?

Reply:
Is this comment not accepted or is there a reply missing?

Reply:
I accepted and made the change.